init.sql

```diff
@@ -118,6 +122,7 @@ CREATE TABLE `file_upload`
     `rating`         enum('A','B','C','D','E') COLLATE utf8_unicode_ci DEFAULT NULL,
     `type`           varchar(50) COLLATE utf8_unicode_ci           DEFAULT NULL,
     `medium`         varchar(50) COLLATE utf8_unicode_ci           DEFAULT NULL,
+    `recording_gain`        int(11) DEFAULT NULL,
```

Comment:
The recording gain has to be added when I upload the sound file!

Reply:
so, the default here is NULL but on production we require a gain value... does this file's code need to be updated @LiuDilongNJ ?